resources/assets/components/pages/AccountPage/Account/AccountNavigation.js

```diff
@@ -3,27 +3,69 @@ import PropTypes from 'prop-types';
 
 import { featureFlag } from '../../../../helpers';
 import NavigationLink from '../../../utilities/NavigationLink/NavigationLink';
+import {
+  EVENT_CATEGORIES,
+  trackAnalyticsEvent,
+} from '../../../../helpers/analytics';
+
+const handleAccountNavTabClick = tabName => {
+  trackAnalyticsEvent(`clicked_account_navlink_${tabName}}`, {
+    action: 'link_clicked',
+    category: EVENT_CATEGORIES.navigation,
+    label: `account_${tabName}`,
+  });
+};
 
 const AccountNavigation = props => (
   <nav className="base-12-grid page-navigation py-3 md:py-6 -no-fade">
     <div className="grid-wide nav-items -mx-3">
-      <NavigationLink exact to="/us/account">
+      <NavigationLink
+        exact
+        to="/us/account"
+        onClick={handleAccountNavTabClick('account')}
```

Comment:
Per https://github.com/DoSomething/phoenix-next/pull/2353#issuecomment-692894379, I'm surprised that changing this to pass a function reference didn't resolve the issue, but I think we should make that change regardless (here and in the `onClick`s below) since otherwise the function would necessarily be invoked for each render.
```suggestion
        onClick={() => handleAccountNavTabClick('account')}
```

Reply:
I went ahead and reverted the function back to having an arrow function. No change 🤔

Reply:
Looks like the preview app is still building based off the new changes, so we won't see the effect yet! Once it finishes building, I'm assuming the issue should be resolved, but... 🤞

Reply:
The page is built now. Also figured out why more than one event was firing at once.I still have the events firing from the `AccountRoute` as well.